gen3/bin/kube-setup-jupyterhub.sh

```diff
@@ -25,7 +25,8 @@ gen3 update_config jupyterhub-config "${GEN3_HOME}/kube/services/jupyterhub/jupy
 
 g3kubectl apply -f "${GEN3_HOME}/kube/services/jupyterhub/serviceaccount.yaml"
 g3kubectl apply -f "${GEN3_HOME}/kube/services/jupyterhub/role-jupyter.yaml"
-g3kubectl apply -f "${GEN3_HOME}/kube/services/jupyterhub/rolebinding-jupyter.yaml"
+namespace="$(gen3 db namespace)"
+g3k_kv_filter ${GEN3_HOME}/kube/services/jupyterhub/rolebinding-jupyter.yaml JUPYTER_BINDING "name: jupyter-binding-$namespace" CURRENT_NAMESPACE "namespace: $namespace" | g3kubectl apply -f -
```

Comment:
are you sure `name: jupyter-binding-$namespace` is correct? a hardcoded `name: jupyter-binding` is what works for me

Reply:
seems working in qa dcp and niaid